packages/gatsby-source-graphql/README.md

```diff
@@ -150,6 +150,49 @@ module.exports = {
 }
 ```
 
+## Composing Apollo Links for production network setup
```

Comment:
Do you have to have Apollo installed or an account set up for this to work?

Reply:
No, this is a pure OSS code: https://www.apollographql.com/docs/link/